.github/workflows/build-and-test.yaml

```diff
@@ -8,29 +9,105 @@ on:
     branches:
       - main
 
-env:
-  product_name: db-operator
-  go_version: "1.18"
-  go_os: linux
-  go_arch: amd64
-  main_go_path: .
-
 jobs:
-  lint: ## https://github.com/golangci/golangci-lint-action#how-to-use
+  build-image:
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set container annotations
```

Comment:
can this until `latest` tag, put into a bash script to cleanup the action and make maybe also the scriting easier to understand?

Reply:
Should look better now